Podfile

```diff
@@ -15,6 +15,9 @@ target 'BTSPocket' do
 # - - - Messages Indicators - - -
   pod 'PKHUD'
   pod 'SwiftMessages'
+  
+# - - - Image cache 'Kingfisher'
+  pod 'Kingfisher', '~> 6.0'
```

Comment:
do we need specific 6.0 version or can we use newest?

Reply:
nop, let me delete it